employees/tests/test_report_detail_and_delete_views.py

```diff
@@ -231,7 +231,6 @@ def test_custom_report_detail_view_should_contains_chosen_project_even_user_is_n
         self.user.projects.remove(self.report.project)
 
         response = self.client.get(self.url)
-
```

Comment:
Actually this blank space wasn't unnecessary because it was separating tested code from assertions which checks whether code runs correctly ;)

Reply:
Added blank space back